### PR TITLE
Add openjdk 9.0.4

### PR DESCRIPTION
**Pull request by MaSven**
Signed-off-by: Sven Marquardt <sven2q@hotmail.de>

	geändert:       src/main/scala/io/sdkman/changelogs/Migrations.scala

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/Migrations.scala [hunk 1]
```diff
@@ -47,4 +47,12 @@ class Migrations {
     insertVersions(CandidateVersion("scala", "2.12.5", Some("UNIVERSAL"), "https://downloads.lightbend.com/scala/2.12.5/scala-2.12.5.zip"))
     updateCandidateDefault("scala", "2.12.5")
   }
+  
+  @ChangeSet(order = "008",id ="008-add_openjdk_9_0_4",author="MaSven")
+  def migrate008(implicit db: MongoDatabase)=insertVersions(
+    CandidateVersion("java","9.0.4-openjdk",Some("MAC_OSX"),"https://download.java.net/java/GA/jdk9/9.0.4/binaries/openjdk-9.0.4_osx-x64_bin.tar.gz"),
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/Migrations.scala]: `CandidateVersion` is now renamed to `Version` for simplicity.

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/Migrations.scala]: Would you be able to move these into the `JavaMigrations` class please? `Migrations` is now deprecated.

**Comment by marc0der** [comment]
You probably need to rebase against the latest master to see the changes that I've made subsequently.

**Comment by MaSven** [comment]
I Migrated to the new classes. Also changed the CandidateVersion to Version.

**Pull request merged**